src/plugins/GUITestBase/src/tests/common_scenarios/Assembling/Assembly_browser/GTTestsAssemblyBrowser.cpp

```diff
@@ -1045,5 +1045,15 @@ GUI_TEST_CLASS_DEFINITION(test_0037) {
     CHECK_SET_ERR(ok, "unexpected clipboard: " + clipboard)
 }
 
+GUI_TEST_CLASS_DEFINITION(test_0038) {
+    // Based on UGENE-7677
+    // 1. open view for _common_data\bam\more_then_100000_reads.bam
+    // Expected state: conversion finished without error
+    GTUtilsDialog::waitForDialog(os, new ImportBAMFileFiller(os));
+    GTFileDialog::openFile(os, testDir + "_common_data/bam/", "more_then_100000_reads.bam");
+    GTUtilsTaskTreeView::waitTaskFinished(os);
+    GTUtilsDialog::checkNoActiveWaiters(os);
```

Comment:
Could you please add a check that 
1) there are no errors in the log 
and 
2) that the active assembly browser window contains expected number of reads?

Otherwise the test will pass even if nothing has happened.

Reply:
OK